deps/rust/cargo.bzl

```
@@ -28,4 +28,5 @@ PACKAGES = WORKERD_CXX_PACKAGES | {
     # tokio is huge, let's enable only features when we actually need them.
     "tokio": crate.spec(version = "1", default_features = False, features = ["net", "rt", "rt-multi-thread", "time"]),
     "tracing": crate.spec(version = "0", default_features = False, features = ["std"]),
+    "bollard": crate.spec(version = "0"),
```

```suggestion
    "bollard": crate.spec(version = "0", default_features = False),
```

This should reduce the dependencies surface and I'm pretty sure you're not uing http or pipe features considering our approach depends on just a custom transport